packages/whats-new/types.d.ts

```diff
@@ -1,3 +1,8 @@
+// Fixed node_modules/@wordpress/components/build-types/dropdown-menu/types.d.ts(43,24): error TS2339: Property 'role' does not exist on type 'HTMLElement'.
+interface HTMLElement {
```

Comment:
Nice, I was actually looking into this error today as well. I've been trying to figure out why this is happening, because `HTMLElement` does appear to have a role element elsewhere.

Reply:
Ah looks like this could be removed after I removed all the `declare module '@wordpress/components` from the repo. Seems to be working without it anyways 👍